config/templates/labels.yaml.envsubst

```diff
@@ -4,6 +4,9 @@ kind: LabelTransformer
 metadata:
   name: labelTransformer
 labels:
+  app.kubernetes.io/instance: '{{ .Release.Name }}'
```

Comment:
This will not work with kustomize 🤔

Reply:
What about a different envsubst file for helm?

https://github.com/kyverno/kyverno/pull/5594/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52

Reply:
That's what I was thinking too.
Either this or some sed magic...